### DISCUSSION TITLE
Cours Mines ParisTech UE12 AP Python

### DISCUSSION THREAD

**Discussion by boisgera** [boisgera/python-fr#8] [General]
Donnez-moi votre identifiant GitHub svp ! (Ainsi que vos nom et prénom si je ne peux pas les déduire des identifiants).

Le mien est @boisgera (Sébastien Boisgérault).

**Comment by timotheedol** [comment]
@timotheedol

**Comment by Florine353** [comment]
@Florine353

**Comment by louislerouge** [comment]
@louislerouge

**Comment by boisgera** [comment]
@MatthieuLam
